easytags.go

```diff
@@ -10,29 +10,33 @@ import (
 	"os"
 	"strings"
 	"unicode"
+	"regexp"
 )
 
 func main() {
 	args := os.Args[1:]
-	tp := "json"
+	var tp []string
+
 	if len(args) < 1 {
 		fmt.Println("Usage : easytags {file_name} {tag_name} \n example: easytags file.go json")
 		return
 	} else if len(args) == 2 {
-		tp = args[1]	
+		provided := strings.Split(args[1], ",")
+		for _, e := range provided {
+			tp = append(tp, strings.TrimSpace(e))
+		}
 	}
-	
+
```

Comment:
This will remove all tags if the second argument is not provided. 
But, existing integrations depend on 'json' to be the default value. 

So, you could check for tp to be nil, and append 'json' as default 
```
 	if tp == nil {
		tp = append(tp, "json")
	}
```

Reply:
For tag removing behavior, you can take a flag like `easytags -r $GOFILE tagnames`
For such switches flag package can be used to parse the command line.

Reply:
Added remove Flag and some refactoring